Sources/XCBProtocol_13_0/Build/SettingsOverrides.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import MessagePack
+import XCBProtocol
+
+public struct SettingsOverrides: Decodable {
+    public let synthesized: [String: String] // e.g. ["TARGET_DEVICE_MODEL": "iPhone12,5", "TARGET_DEVICE_OS_VERSION": "13.3"]
```

Comment:
not for this PR, but it would be nice to start converting some of these comments to doc strings.